tests/Unit/Visualization/Python/Test_ApplyPointwise.py

```diff
@@ -0,0 +1,59 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+from spectre.Visualization.ApplyPointwise import (snake_case_to_camel_case,
+                                                  Kernel, apply_pointwise,
+                                                  apply_pointwise_command)
+
+import numpy as np
+import numpy.testing as npt
+import os
+import shutil
+import spectre.IO.H5 as spectre_h5
+import unittest
+from click.testing import CliRunner
+from spectre.Informer import unit_test_src_path, unit_test_build_path
+from spectre.DataStructures import DataVector
+from spectre.DataStructures.Tensor import Scalar
+
+
+def psi_squared(psi: Scalar[DataVector]) -> Scalar[DataVector]:
+    return Scalar[DataVector](np.array(psi)**2)
+
+
+class TestApplyPointwise(unittest.TestCase):
+    def setUp(self):
+        self.test_dir = os.path.join(unit_test_build_path(),
+                                     'Visualization/ApplyPointwise')
+        self.h5_filename = os.path.join(self.test_dir, "Test.h5")
+        os.makedirs(self.test_dir, exist_ok=True)
+        shutil.copyfile(
+            os.path.join(unit_test_src_path(),
+                         'Visualization/Python/VolTestData0.h5'),
+            self.h5_filename)
+
+    def tearDown(self):
+        shutil.rmtree(self.test_dir)
+
+    def test_snake_case_to_camel_case(self):
+        self.assertEqual(snake_case_to_camel_case("hello_world"), "HelloWorld")
+
+    def test_apply_pointwise(self):
+        open_h5_files = [spectre_h5.H5File(self.h5_filename, "a")]
+        open_volfiles = [
+            h5file.get_vol("/element_data") for h5file in open_h5_files
+        ]
+
+        kernel = Kernel(psi_squared)
```

Comment:
Is there an easy way to test something like the shift magnitude where multiple components need to be retrieved and combined?

Reply:
added another kernel, please see fixup